libraries/chain/webassembly/runtimes/eos-vm.cpp

```diff
@@ -6,6 +6,7 @@
 #include <eosio/chain/wasm_eosio_constraints.hpp>
 //eos-vm includes
 #include <eosio/vm/backend.hpp>
+#include "host_function_registrator.cpp"
```

Comment:
I'm fine with including at this layer.  But, I am strongly against including a cpp file.  So, just move this back to the includes and include like normal and rename to .hpp.